src/app/server/Server.h

```diff
@@ -324,15 +343,24 @@ class Server
                 {
                     while (count)
                     {
-                        Access::GetAccessControl().DeleteEntry(nullptr, fabricIndex, --count);
+                        (void) Access::GetAccessControl().DeleteEntry(nullptr, fabricIndex, --count);
```

Comment:
should we log here on failure as well instead of casting CHIP_ERROR to void and ignoring?

Reply:
Changes nothing. Behavior not changed, just made more explicit.